src/frontends/tensorflow/src/graph_iterator_saved_model.hpp

```diff
@@ -175,6 +179,19 @@ class GraphIteratorSavedModel : public GraphIteratorProto {
             for (const auto& output : serving_default->second->outputs()) {
                 (*m_outputs_map)[output.second.name()] = output.first;
             }
+        } else if (validSignatures.size() > 0) {
```

Comment:
Do we need to add similar logic to TF Python decoder?

Reply:
Indeed, we currently have some misalignment between conversion from memory and conversion from disk for such models with non-default signatures. Models converted from memory will represent the first non-default signature, models converted from disk will represent all non-default signatures.

In order to fix this misalignment for conversion from memory, it requires normal effort: convert several Concrete functions for each signature, and concatenate them (but they may have duplication in graph nodes). But conversion from memory has advantages that user can select exact signature (ConcreteFunction) to convert that is absent option for conversion from disk. So we will have to cover all signatures.

I think at this point, it makes sense to document conversion of non-default signatures and explain how to work with it.